board/ipaccess/ipa9131/sec.c

```diff
@@ -0,0 +1,43 @@
+#include <common.h>
+#include <compiler.h>
+#include <asm/errno.h>
+#include <asm/io.h>
+#include <asm/immap_85xx.h>
+#include "characterisation.h"
+
+
+#define sec_out_be32(x,y) out_be32((volatile unsigned __iomem *)(x),(y))
+
+#define SECMON_HPCOMR        (CONFIG_SYS_IMMR + 0x000e6004)
+
+
+
+static void set_sec_state(void)
```

Comment:
Does this have to be three writes to the same register, or can we do:

```c
    u32 regval = 0x3;
    if (characterisation_is_specials_mode())
        regval |= 0x200;
    sec_out_be32(SECMON_HPCOMR, regval);
```

I'm just curious about this - don't go spending too much time on this...